docs/content/manual/_incoming/2722_VMs_can_not_start_if_a_node_contains_more_than_60_VMs.md

```diff
@@ -0,0 +1,27 @@
+---
+title: VMs can't start if a node contains more than ~60 VMs
+category: console
+tags: VM, p1, acceptance
+---
+Ref: https://github.com/harvester/harvester/issues/2722
+
+![image](https://user-images.githubusercontent.com/5169694/192251104-7a53a1a9-260d-4e90-aade-1b3e7c11cc52.png)
+
+
+### Verify Steps:
+1. Install Harvester with any nodes
```

Comment:
Provisioning 60vms, do we need a "node" that has a certain amount of specs?  Should this be a "single-node" since we want to ensure that the Node is respecting the limit?

Reply:
The root cause of the issue is pods amount, so even single node which able to create VMs over than 60 still hit the criteria.

Reply:
Ahhh! Thanks for helping me understand that a bit more :+1: :smile: